redbot/core/bot.py

```diff
@@ -24,7 +22,7 @@
 from .sentry import SentryManager
 
 
-class RedBase(BotBase, RPCMixin):
+class RedBase(commands.GroupMixin, commands.bot.BotBase, RPCMixin):
```

Comment:
Why call our own reference to `BotBase`? I don't see a whole lot of sense in doing this

Reply:
See [here](https://github.com/Cog-Creators/Red-DiscordBot/compare/V3/develop...Tobotimus:V3/refactor_cmd_decorators_2?expand=1#diff-b4e8893536462f5cd5d7e291288c4d5eR25) for an example of what I mean, I don't really think its necessary but all of this inheritance stuff is pretty confusing to some, perhaps this makes it a bit simpler? I'm not sure.

Reply:
Sorry, [this](https://github.com/Tobotimus/Red-DiscordBot/compare/V3/refactor_cmd_decorators...Tobotimus:V3/refactor_cmd_decorators_2?expand=1#diff-b4e8893536462f5cd5d7e291288c4d5eR25) is probably a better way to show the example